CHANGELOG.md

```diff
@@ -24,6 +24,8 @@
   which they returned previously.
 - Fix issue #244: Collector to keep querying. Remove the parameter for maximum allowed
   times a gRPC call can fail and keeps `node-collector` querying forever.
+- Fix the issue #259: Expose foundation account address. The account information now can
+  be queried by using an account index as well.
```

Comment:
```suggestion
- `GetAccountInfo` endpoint supports querying the account via the account index.
```

I missed that in the previous commit you did, but if you look at the changelogs we have, they do not mention specific github issues. SO please keep them like that.